app/controllers/manager_profiles_controller.rb

```diff
@@ -36,6 +37,11 @@ def remove_staff
   end
 
   def manage_staffs
+    @roles = {}
+    EventStaff.roles.each do |key, value|
```

Comment:
Unused block argument - `value`. If it's necessary, use `_` or `_value` as an argument name to indicate that it won't be used.